module-1/web-project/your-code/Web scraping.ipynb

```diff
@@ -0,0 +1,1005 @@
+{
```

Comment:
Amazing job Miriam in the Web Scraping section. We can try to modify your searching and selecting functions to grant that you take the event and the associated meetup group instead of the next or previous one. If you have time we can take a look on it.